src/SelfServe/SelfServeTypes.ts

```diff
@@ -113,18 +130,27 @@ export interface SmartUiInput {
   disabled?: boolean;
 }
 
-export enum SelfServeNotificationType {
-  info = "info",
-  warning = "warning",
-  error = "error",
+export enum PortalNotificationType {
```

Comment:
>PortalNotificationType [](start = 12, length = 22)

Status. to me, these dont look like types but statuses.

Reply:
not needed anymore